app/experimenter/experiments/tests/factories.py

```diff
@@ -75,7 +75,10 @@ class ExperimentFactory(
     pref_branch = factory.LazyAttribute(
         lambda o: random.choice(Experiment.PREF_BRANCH_CHOICES[1:])[0]
     )
-    firefox_version = factory.LazyAttribute(
+    firefox_min_version = factory.LazyAttribute(
+        lambda o: random.choice(Experiment.VERSION_CHOICES[1:])[0]
+    )
+    firefox_max_version = factory.LazyAttribute(
```

Comment:
Actually it would be good if we also threw in a case that this is sometimes randomly empty so we stress test the cases where it's not set.

Reply:
I think if you just drop the [1:] slice then this should randomly be None sometimes which should cover it.